### DISCUSSION TITLE
Welcome to marginfi-v2 Discussions!

### DISCUSSION THREAD

**Discussion by edgarpavlovsky** [mrgnlabs/marginfi-v2#101] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by holooeKT** [comment]
我的marginfi为什么邮箱登录不上，然后代币也取不出来，是怎么回事，是不是骗子？
